app/views/exam_templates/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for exam_template,
+             as: :exam_template,
```

Comment:
You shouldn't need the `as` or `url` keys (rails is pretty good about defaults).